chapters/ch07.asciidoc

```diff
@@ -1415,7 +1431,7 @@ cast(`a`, `b`)
 // <- [`a`, `b`]
 ----
 
-You may also want to cast +NodeList+ DOM element collections, like those returned from +document.querySelectorAll+, through the spread operator. Once again, this is made possible thanks to ES6 adding conformance to the iterator protocol for +NodeList+.
+You may also want to cast +NodeList+ DOM element collections, like those returned from +document.querySelectorAll+, through the spread operator. This can be helpful when we need access to native array methods like +Array#map+ or +Array#filter+. Once again, this is made possible thanks to ES6 adding conformance to the iterator protocol for +NodeList+.
```

Comment:
This makes it sound as if ES6 directly changed `NodeList`.

1. ES6 introduced the iterator protocol
2. WebIDL embraced it (adding `iterable`)
3. Non-ES web standards such as the DOM Standard started to use `iterable`, e.g. for `NodeList`

That’s probably too much detail for the book, but perhaps this could be rephrased a bit? Something like:

> This is possible because the DOM Standard made `NodeList`s iterable after ES6 defined the iterator protocol.